portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,8 +40,10 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int commentLimit = Integer.parseInt(request.getParameter("limit"));
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    String languageCode = request.getParameter("languageCode");
+    System.out.println(languageCode);
```

Comment:
Is this used for debugging? Remove?

Reply:
The println was used for debugging, I removed it in another commit.